resources/analysisTools/copyNumberEstimationWorkflow/estimateHRDScore.sh

```diff
@@ -76,7 +76,7 @@ do
 	${RSCRIPT_BINARY} ${TOOL_HRD_ESTIMATION} \
 		 $combProFileNoArtifacts \
 		 $combProFile.tmp \
-		 $gender \
+		 $patientsex \
```

Comment:
This change was done by me but it wasn't necessary, can we revert it?

Reply:
Sure, it will stay in history, so everyone can see it, but we can alter it in a new commit. So no problem.